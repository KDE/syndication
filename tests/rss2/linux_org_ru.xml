<?xml version="1.0" encoding="utf-8"?>
<!-- html detection bug 168864 -->






<rss version="2.0">
<channel>
<link>http://www.linux.org.ru/</link>
<language>ru</language>

<title>Linux.org.ru: Новости</title><pubDate>Wed, 15 Oct 2008 23:20:41 +0400</pubDate><description>Linux.org.ru: Новости</description>
<item>
  <title>Русификация интерфейса GeoServer</title>
  <author>simgislab</author>
  <link>http://www.linux.org.ru/jump-message.jsp?msgid=3170374</link>
  <guid>http://www.linux.org.ru/jump-message.jsp?msgid=3170374</guid>
  <pubDate>Wed, 15 Oct 2008 05:20:29 +0400</pubDate>
  <description>
	На GIS-Lab.info закончена русификация GeoServer, написанного на Java и активно развивающегося открытого картографического web-сервера, поддерживающего спецификации OGC: WMS, WCS, WFS, WFS-T.</description>
</item><item>
  <title>Java ME SDK 3.0</title>
  <author>iZEN</author>
  <link>http://www.linux.org.ru/jump-message.jsp?msgid=3168576</link>
  <guid>http://www.linux.org.ru/jump-message.jsp?msgid=3168576</guid>
  <pubDate>Tue, 14 Oct 2008 13:47:43 +0400</pubDate>
  <description>
	Вышел Java Platform Micro Edition Software Development Kit 3.0
&lt;br&gt;
&lt;br&gt;Новшества:
&lt;ul&gt;
&lt;li&gt;Интеграция со сторонними эмуляторами и устройствами Windows Mobile
&lt;li&gt;Прямая загрузка кода на устройство и непосредственная отладка на нём
&lt;li&gt;Поддерживаемые конфигурации и профили: CLDC/MIDP, CDC/FP/PBP/AGUI и BD-J
&lt;li&gt;Новая виртуальная машина CLDC HotSpot
&lt;li&gt;Оптимизиция стэка MSA 1.1 с расширениями
&lt;li&gt;Поддержка профилирования
&lt;li&gt;Поддержка окружения BD-J
&lt;li&gt;Новая среда разработки на основе Netbeans Platform
&lt;li&gt;Интегрирован легковесный UI Toolkit (LWUIT)
&lt;li&gt;Интегрированная в SDK база данных по устройствам
&lt;/ul&gt;
Минимальные системные требования:
&lt;ul&gt;
&lt;li&gt;200 МБ на жёстком диске
&lt;li&gt;1 ГБ ОЗУ
&lt;li&gt;процессор 1 ГГц
&lt;li&gt;Java SE Development Kit (JDK 1.6)
&lt;li&gt;Apple QuickTime player (для воспроизведения звуков формата AMR под Windows)
&lt;/ul&gt;Для программирования в BD-J (Blu-ray Java):
&lt;ul&gt;
&lt;li&gt;CyberLink Power DVD (свободно скачивается)
&lt;li&gt;ArcSoft TotalMedia Theatre или другой плеер, способный проигрывать диски Blu-ray.
&lt;/ul&gt;</description>
</item><item>
  <title>Имитационное моделирование в MapSim</title>
  <author>dave</author>
  <link>http://www.linux.org.ru/jump-message.jsp?msgid=3170711</link>
  <guid>http://www.linux.org.ru/jump-message.jsp?msgid=3170711</guid>
  <pubDate>Wed, 15 Oct 2008 11:00:04 +0400</pubDate>
  <description>
	По свободной лицензии LGPL вышел симулятор MapSim, который позволяет моделировать динамические системы. MapSim использует собственный высоко- уровневый язык моделирования, который поддерживает многие стандартные математические функции, а также оперирует такими элементами как интегралы (резервуары), конвейеры, потоки и т.п. Этот язык подходит для описания задач системной динамики (System Dynamics).
&lt;p&gt;В комплект входит библиотека, написанная на .NET/Mono. Есть очень простой клиент для WinForms, который запускается в Windows и Linux. Этот клиент позволяет загружать модели, интегрировать их, а затем сохранять результаты симуляции в виде CSV файла. 
&lt;p&gt;В реальности возможности библиотеки гораздо шире – ее использует полноценная среда моделирования с удобным графическим интерфейсом. Поэтому приветствуются желающие доработать существующий WinForms клиент. Также очень интересен порт клиента на Gtk#, чтобы приложение выглядело как родное для среды Gnome.</description>
</item><item>
  <title>Вышел Lazarus 0.9.26</title>
  <author>sniper21</author>
  <link>http://www.linux.org.ru/jump-message.jsp?msgid=3170676</link>
  <guid>http://www.linux.org.ru/jump-message.jsp?msgid=3170676</guid>
  <pubDate>Wed, 15 Oct 2008 10:32:26 +0400</pubDate>
  <description>
	После 11 месяцев разработки вышла новая версия RAD IDE для языка FreePascal.
&lt;p&gt;С версии 0.9.24 сделано 3973 изменений и 703 ошибок исправлено.
&lt;ul&gt;
&lt;li&gt;Сильно улучшены наборы виджетов Carbon и QT (&lt;a href=&quot;http://www.linux.org.ru/view-message.jsp?msgid=2784910&quot;&gt;http://www.linux.org.ru/view-message....&lt;/a&gt;)
&lt;li&gt;Полная поддерка Unicode: добавлены полезные функции UTF8Copy, UTF8UpperCase, UTF8LowerCase в модуль LCLProc
&lt;li&gt;Также улучшена поддержка библиотеки GTK2
&lt;li&gt;новые свойства TForm.LCLVersion и TFrame.LCLVersion
&lt;li&gt;удалены стары флаги компилятора (-Sp -S2 -St -So)
&lt;li&gt;Новые иконки
&lt;/ul&gt;
&lt;p&gt;Также недавно была выпущена версия FreePascal 2.2.2 с исправлениями ошибок.
&lt;p&gt;&gt;&gt;&gt; &lt;a href=&quot;http://sourceforge.net/project/showfiles.php?group_id=89339&quot;&gt;Скачать&lt;/a&gt;.</description>
</item><item>
  <title>Maemo - &amp;quot;EASY-DEBIAN&amp;quot; теперь доступен из Extras Repository</title>
  <author>lester</author>
  <link>http://www.linux.org.ru/jump-message.jsp?msgid=3170389</link>
  <guid>http://www.linux.org.ru/jump-message.jsp?msgid=3170389</guid>
  <pubDate>Wed, 15 Oct 2008 05:39:17 +0400</pubDate>
  <description>
	&quot;EASY-DEBIAN&quot; - это набор средств для использования всей мощи дебиана на &quot;таблетках&quot; от нокии. По-сути это просто образ дебиана, который находится на флешке в виде отдельного файла, установка, настройка и использование которого предельно облегчены для пользователя.
&lt;br&gt;
&lt;br&gt;Из важных изменений:
&lt;ul&gt;
&lt;li&gt;теперь вы можете установить его из &quot;Диспетчера приложений&quot;, достаточно обновить список доступных пакетов
&lt;li&gt;значительно возросла скорость работы
&lt;li&gt;в базовую установку добавлен LXDE
&lt;li&gt;добавлена поддержка режима &quot;Performance Mode&quot; для процессора, предназначенного для более комфортной работы с OpenOffice и IceWeasel
&lt;/ul&gt;</description>
</item><item>
  <title>Школы Новой Зеландии переходят на свободное програмное обеспечение</title>
  <author>G-UNiX</author>
  <link>http://www.linux.org.ru/jump-message.jsp?msgid=3168814</link>
  <guid>http://www.linux.org.ru/jump-message.jsp?msgid=3168814</guid>
  <pubDate>Tue, 14 Oct 2008 15:18:23 +0400</pubDate>
  <description>
	Министерство образования Новой Зеландии отказалось оплачивать лицензии компании Microsoft и разработало план перехода на свободное программное обеспечение, на базе GNU/Linux, до 2010 года. Полгода назад Министерство образования Новой Зеландии не стало покупать пакет лицензий от Microsoft, а переадресовало средства на миграцию на GNU/Linux, что позволило забыть о расходах на лицензирование. Министерство изменило и планирование бюджета, которое, теперь, направлено на развитие образовательной среды на платформе GNU/Linux и обратило внимание школ на то, что не стоит платить большие деньги за неэффективное программное обеспечение и зацикливаться на одной платформе, а лучше повышать эффективность применения программного обеспечения в учебном процессе и качество образования. Ричард Столлман (Richard Stallman), совершаюший турне по Новой Зеландии, во время обмена мнениями с новозеландскими школами, посоветовал использовать программы группового образования, как, например, WikiEducator, а также создавать образовательный контент, доступный школьникам не только для использования, но и для редактирования.</description>
</item><item>
  <title>Dillo 2.0</title>
  <author>mad_soft</author>
  <link>http://www.linux.org.ru/jump-message.jsp?msgid=3169429</link>
  <guid>http://www.linux.org.ru/jump-message.jsp?msgid=3169429</guid>
  <pubDate>Tue, 14 Oct 2008 19:28:38 +0400</pubDate>
  <description>
	После более двух лет, прошедших с выхода предыдущего релиза, наконец-то вышла новая версия сверхлёгкого браузера Dillo. Как известно, его разработка была заморожена в 2007-м году, но в начале этого года возобновилась, и результат не замедлил себя ждать. Главные изменения коснулись интерфейса - он был полностью переписан с GTK1 на FLTK2, что позволило сократить вдвое требования к памяти, а также реализовать поддержку UTF-8. Кроме того, в списке нововведений присутствуют табы, сглаживание шрифтов и прочие вкусности.
&lt;br&gt;
&lt;br&gt;Анонсируя новую версию, авторы Dillo призывают всех заинтересовавшихся разработчиков помочь и присоединиться к проекту.
&lt;br&gt;
&lt;br&gt;Полный Changelog: &lt;a href=&quot;http://cvs.auriga.wearlab.de/cgi-bin/cvsweb.cgi/dillo2/ChangeLog?rev=1.133;content-type=text%2Fplain;cvsroot=dillo&quot;&gt;http://cvs.auriga.wearlab.de/cgi-bin/...&lt;/a&gt;</description>
</item><item>
  <title>Debian-Installer (D-I) и Custom Debian Distributions (CDD) : Промышленная платформа для создания специализированных дистрибутивов и сред инсталяции </title>
  <author>IBM_dW</author>
  <link>http://www.linux.org.ru/jump-message.jsp?msgid=3168633</link>
  <guid>http://www.linux.org.ru/jump-message.jsp?msgid=3168633</guid>
  <pubDate>Tue, 14 Oct 2008 14:15:21 +0400</pubDate>
  <description>
	Статья представляет собой обзор архитектуры и методов применения инфраструктуры проекта Debian в коммерческих и свободных разработках. Посвящена построению сред инсталяции, изменению поведения инсталятора, распределённой установке, сборке LiveCD, добавлению стороннего ПО.
Первоначально представлена в виде доклада на 4й международной конференции разработчиков и пользователей свободного программного обеспечения Linux Vacation Eastern Europe'2008 (LVEE-2008) 28 июня 2008 года, Гродно, Беларусь.
Позже был дополнен новой информацией о работах компании Promwad в области использования Debian-based дистрибутивов. </description>
</item><item>
  <title>Вышел linphone 3.0.0</title>
  <author>Outspector</author>
  <link>http://www.linux.org.ru/jump-message.jsp?msgid=3167477</link>
  <guid>http://www.linux.org.ru/jump-message.jsp?msgid=3167477</guid>
  <pubDate>Mon, 13 Oct 2008 21:07:58 +0400</pubDate>
  <description>
	Вышла новая версия IP-телефона linphone. Телефон имеет консольный и графический интерфейс, последний построен на библиотеке GTK.
&lt;br&gt;
&lt;br&gt;Телефон поддерживает работу по протоколу SIP, кодирование аудио кодеками (speex, G711, GSM и iLBC) и поддерживает работу с веб-камерами и кодирование видео (H263-1998, MPEG4, theora и H264). Также поддерживается обмен текстовыми сообщениями с использованием протокола SIMPLE.
&lt;p&gt;Полный список возможностей здесь: &lt;a href=&quot;http://linphone.org/index.php/eng/features&quot;&gt;http://linphone.org/index.php/eng/fea...&lt;/a&gt;.
&lt;br&gt;
&lt;br&gt;Довольно скупая новость на главной странице гласит, что новая версия linphone получила:
&lt;ul&gt;
&lt;li&gt;переработанный графический интерфейс,
&lt;li&gt;улучшенную работу с видео,
&lt;li&gt;улучшенную работу для тех, кто сидит за файрволом,
&lt;/ul&gt;и еще ряд не столь значительных изменений.</description>
</item><item>
  <title>Новый дистрибутив для Openmoko - FDOM</title>
  <author>JamesBond007</author>
  <link>http://www.linux.org.ru/jump-message.jsp?msgid=3166880</link>
  <guid>http://www.linux.org.ru/jump-message.jsp?msgid=3166880</guid>
  <pubDate>Mon, 13 Oct 2008 17:30:51 +0400</pubDate>
  <description>
	FDOM - это новый дистрибутив GNU/Linux, созданный для телефонов Neo FreeRunner.
&lt;p&gt;Создатель этого дистрибутива (David-Reyes Samblas Martinez) поставил перед собой цель собрать воедино все существующие *и интересные* приложения для Openmoko и тем самым увеличить его функциональность.
&lt;p&gt;В состав ОС входят: различные веб-браузеры (легковесный Mozilla Minimo и не очень легковесный Firefox), Pidgin в роли Instant Messaging клиента, MPlayer с надстройкой pythm на Python и GTK+, различные мультимедиа кодеки (a52m, aalib, avcodec, faac, faad2, ffmpeg, OGG, FLAC etc), TangoGPS — GPS в связке с картами Google, в роли оконного менеджера Enlightement, разнообразные эмуляторы терминала и командные интерпретаторы (включая zsh!), кучи нативных игрушек и прочие вкусности.</description>
</item>

</channel>
</rss>
