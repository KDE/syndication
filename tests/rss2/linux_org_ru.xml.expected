# Feed begin ######################
title: #Linux.org.ru: Новости#
link: #http://www.linux.org.ru/#
description: #Linux.org.ru: Новости#
language: #ru#
# Item begin ######################
id: #http://www.linux.org.ru/jump-message.jsp?msgid=3170374#
title: #Русификация интерфейса GeoServer#
link: #http://www.linux.org.ru/jump-message.jsp?msgid=3170374#
description: #На GIS-Lab.info закончена русификация GeoServer, написанного на Java и активно развивающегося открытого картографического web-сервера, поддерживающего спецификации OGC: WMS, WCS, WFS, WFS-T.#
datePublished: #Wed Oct 15 03:20:29 2008#
dateUpdated: #Wed Oct 15 03:20:29 2008#
# Person begin ####################
name: #simgislab#
# Person end ######################
# Item end ########################
# Item begin ######################
id: #http://www.linux.org.ru/jump-message.jsp?msgid=3168576#
title: #Java ME SDK 3.0#
link: #http://www.linux.org.ru/jump-message.jsp?msgid=3168576#
description: #Вышел Java Platform Micro Edition Software Development Kit 3.0
<br>
<br>Новшества:
<ul>
<li>Интеграция со сторонними эмуляторами и устройствами Windows Mobile
<li>Прямая загрузка кода на устройство и непосредственная отладка на нём
<li>Поддерживаемые конфигурации и профили: CLDC/MIDP, CDC/FP/PBP/AGUI и BD-J
<li>Новая виртуальная машина CLDC HotSpot
<li>Оптимизиция стэка MSA 1.1 с расширениями
<li>Поддержка профилирования
<li>Поддержка окружения BD-J
<li>Новая среда разработки на основе Netbeans Platform
<li>Интегрирован легковесный UI Toolkit (LWUIT)
<li>Интегрированная в SDK база данных по устройствам
</ul>
Минимальные системные требования:
<ul>
<li>200 МБ на жёстком диске
<li>1 ГБ ОЗУ
<li>процессор 1 ГГц
<li>Java SE Development Kit (JDK 1.6)
<li>Apple QuickTime player (для воспроизведения звуков формата AMR под Windows)
</ul>Для программирования в BD-J (Blu-ray Java):
<ul>
<li>CyberLink Power DVD (свободно скачивается)
<li>ArcSoft TotalMedia Theatre или другой плеер, способный проигрывать диски Blu-ray.
</ul>#
datePublished: #Tue Oct 14 11:47:43 2008#
dateUpdated: #Tue Oct 14 11:47:43 2008#
# Person begin ####################
name: #iZEN#
# Person end ######################
# Item end ########################
# Item begin ######################
id: #http://www.linux.org.ru/jump-message.jsp?msgid=3170711#
title: #Имитационное моделирование в MapSim#
link: #http://www.linux.org.ru/jump-message.jsp?msgid=3170711#
description: #По свободной лицензии LGPL вышел симулятор MapSim, который позволяет моделировать динамические системы. MapSim использует собственный высоко- уровневый язык моделирования, который поддерживает многие стандартные математические функции, а также оперирует такими элементами как интегралы (резервуары), конвейеры, потоки и т.п. Этот язык подходит для описания задач системной динамики (System Dynamics).
<p>В комплект входит библиотека, написанная на .NET/Mono. Есть очень простой клиент для WinForms, который запускается в Windows и Linux. Этот клиент позволяет загружать модели, интегрировать их, а затем сохранять результаты симуляции в виде CSV файла. 
<p>В реальности возможности библиотеки гораздо шире – ее использует полноценная среда моделирования с удобным графическим интерфейсом. Поэтому приветствуются желающие доработать существующий WinForms клиент. Также очень интересен порт клиента на Gtk#, чтобы приложение выглядело как родное для среды Gnome.#
datePublished: #Wed Oct 15 09:00:04 2008#
dateUpdated: #Wed Oct 15 09:00:04 2008#
# Person begin ####################
name: #dave#
# Person end ######################
# Item end ########################
# Item begin ######################
id: #http://www.linux.org.ru/jump-message.jsp?msgid=3170676#
title: #Вышел Lazarus 0.9.26#
link: #http://www.linux.org.ru/jump-message.jsp?msgid=3170676#
description: #После 11 месяцев разработки вышла новая версия RAD IDE для языка FreePascal.
<p>С версии 0.9.24 сделано 3973 изменений и 703 ошибок исправлено.
<ul>
<li>Сильно улучшены наборы виджетов Carbon и QT (<a href="http://www.linux.org.ru/view-message.jsp?msgid=2784910">http://www.linux.org.ru/view-message....</a>)
<li>Полная поддерка Unicode: добавлены полезные функции UTF8Copy, UTF8UpperCase, UTF8LowerCase в модуль LCLProc
<li>Также улучшена поддержка библиотеки GTK2
<li>новые свойства TForm.LCLVersion и TFrame.LCLVersion
<li>удалены стары флаги компилятора (-Sp -S2 -St -So)
<li>Новые иконки
</ul>
<p>Также недавно была выпущена версия FreePascal 2.2.2 с исправлениями ошибок.
<p>>>> <a href="http://sourceforge.net/project/showfiles.php?group_id=89339">Скачать</a>.#
datePublished: #Wed Oct 15 08:32:26 2008#
dateUpdated: #Wed Oct 15 08:32:26 2008#
# Person begin ####################
name: #sniper21#
# Person end ######################
# Item end ########################
# Item begin ######################
id: #http://www.linux.org.ru/jump-message.jsp?msgid=3170389#
title: #Maemo - &quot;EASY-DEBIAN&quot; теперь доступен из Extras Repository#
link: #http://www.linux.org.ru/jump-message.jsp?msgid=3170389#
description: #&quot;EASY-DEBIAN&quot; - это набор средств для использования всей мощи дебиана на &quot;таблетках&quot; от нокии. По-сути это просто образ дебиана, который находится на флешке в виде отдельного файла, установка, настройка и использование которого предельно облегчены для пользователя.
<br>
<br>Из важных изменений:
<ul>
<li>теперь вы можете установить его из &quot;Диспетчера приложений&quot;, достаточно обновить список доступных пакетов
<li>значительно возросла скорость работы
<li>в базовую установку добавлен LXDE
<li>добавлена поддержка режима &quot;Performance Mode&quot; для процессора, предназначенного для более комфортной работы с OpenOffice и IceWeasel
</ul>#
datePublished: #Wed Oct 15 03:39:17 2008#
dateUpdated: #Wed Oct 15 03:39:17 2008#
# Person begin ####################
name: #lester#
# Person end ######################
# Item end ########################
# Item begin ######################
id: #http://www.linux.org.ru/jump-message.jsp?msgid=3168814#
title: #Школы Новой Зеландии переходят на свободное програмное обеспечение#
link: #http://www.linux.org.ru/jump-message.jsp?msgid=3168814#
description: #Министерство образования Новой Зеландии отказалось оплачивать лицензии компании Microsoft и разработало план перехода на свободное программное обеспечение, на базе GNU/Linux, до 2010 года. Полгода назад Министерство образования Новой Зеландии не стало покупать пакет лицензий от Microsoft, а переадресовало средства на миграцию на GNU/Linux, что позволило забыть о расходах на лицензирование. Министерство изменило и планирование бюджета, которое, теперь, направлено на развитие образовательной среды на платформе GNU/Linux и обратило внимание школ на то, что не стоит платить большие деньги за неэффективное программное обеспечение и зацикливаться на одной платформе, а лучше повышать эффективность применения программного обеспечения в учебном процессе и качество образования. Ричард Столлман (Richard Stallman), совершаюший турне по Новой Зеландии, во время обмена мнениями с новозеландскими школами, посоветовал использовать программы группового образования, как, например, WikiEducator, а также создавать образовательный контент, доступный школьникам не только для использования, но и для редактирования.#
datePublished: #Tue Oct 14 13:18:23 2008#
dateUpdated: #Tue Oct 14 13:18:23 2008#
# Person begin ####################
name: #G-UNiX#
# Person end ######################
# Item end ########################
# Item begin ######################
id: #http://www.linux.org.ru/jump-message.jsp?msgid=3169429#
title: #Dillo 2.0#
link: #http://www.linux.org.ru/jump-message.jsp?msgid=3169429#
description: #После более двух лет, прошедших с выхода предыдущего релиза, наконец-то вышла новая версия сверхлёгкого браузера Dillo. Как известно, его разработка была заморожена в 2007-м году, но в начале этого года возобновилась, и результат не замедлил себя ждать. Главные изменения коснулись интерфейса - он был полностью переписан с GTK1 на FLTK2, что позволило сократить вдвое требования к памяти, а также реализовать поддержку UTF-8. Кроме того, в списке нововведений присутствуют табы, сглаживание шрифтов и прочие вкусности.
<br>
<br>Анонсируя новую версию, авторы Dillo призывают всех заинтересовавшихся разработчиков помочь и присоединиться к проекту.
<br>
<br>Полный Changelog: <a href="http://cvs.auriga.wearlab.de/cgi-bin/cvsweb.cgi/dillo2/ChangeLog?rev=1.133;content-type=text%2Fplain;cvsroot=dillo">http://cvs.auriga.wearlab.de/cgi-bin/...</a>#
datePublished: #Tue Oct 14 17:28:38 2008#
dateUpdated: #Tue Oct 14 17:28:38 2008#
# Person begin ####################
name: #mad_soft#
# Person end ######################
# Item end ########################
# Item begin ######################
id: #http://www.linux.org.ru/jump-message.jsp?msgid=3168633#
title: #Debian-Installer (D-I) и Custom Debian Distributions (CDD) : Промышленная платформа для создания специализированных дистрибутивов и сред инсталяции#
link: #http://www.linux.org.ru/jump-message.jsp?msgid=3168633#
description: #Статья представляет собой обзор архитектуры и методов применения инфраструктуры проекта Debian в коммерческих и свободных разработках. Посвящена построению сред инсталяции, изменению поведения инсталятора, распределённой установке, сборке LiveCD, добавлению стороннего ПО.
Первоначально представлена в виде доклада на 4й международной конференции разработчиков и пользователей свободного программного обеспечения Linux Vacation Eastern Europe&apos;2008 (LVEE-2008) 28 июня 2008 года, Гродно, Беларусь.
Позже был дополнен новой информацией о работах компании Promwad в области использования Debian-based дистрибутивов.#
datePublished: #Tue Oct 14 12:15:21 2008#
dateUpdated: #Tue Oct 14 12:15:21 2008#
# Person begin ####################
name: #IBM_dW#
# Person end ######################
# Item end ########################
# Item begin ######################
id: #http://www.linux.org.ru/jump-message.jsp?msgid=3167477#
title: #Вышел linphone 3.0.0#
link: #http://www.linux.org.ru/jump-message.jsp?msgid=3167477#
description: #Вышла новая версия IP-телефона linphone. Телефон имеет консольный и графический интерфейс, последний построен на библиотеке GTK.
<br>
<br>Телефон поддерживает работу по протоколу SIP, кодирование аудио кодеками (speex, G711, GSM и iLBC) и поддерживает работу с веб-камерами и кодирование видео (H263-1998, MPEG4, theora и H264). Также поддерживается обмен текстовыми сообщениями с использованием протокола SIMPLE.
<p>Полный список возможностей здесь: <a href="http://linphone.org/index.php/eng/features">http://linphone.org/index.php/eng/fea...</a>.
<br>
<br>Довольно скупая новость на главной странице гласит, что новая версия linphone получила:
<ul>
<li>переработанный графический интерфейс,
<li>улучшенную работу с видео,
<li>улучшенную работу для тех, кто сидит за файрволом,
</ul>и еще ряд не столь значительных изменений.#
datePublished: #Mon Oct 13 19:07:58 2008#
dateUpdated: #Mon Oct 13 19:07:58 2008#
# Person begin ####################
name: #Outspector#
# Person end ######################
# Item end ########################
# Item begin ######################
id: #http://www.linux.org.ru/jump-message.jsp?msgid=3166880#
title: #Новый дистрибутив для Openmoko - FDOM#
link: #http://www.linux.org.ru/jump-message.jsp?msgid=3166880#
description: #FDOM - это новый дистрибутив GNU/Linux, созданный для телефонов Neo FreeRunner.
<p>Создатель этого дистрибутива (David-Reyes Samblas Martinez) поставил перед собой цель собрать воедино все существующие *и интересные* приложения для Openmoko и тем самым увеличить его функциональность.
<p>В состав ОС входят: различные веб-браузеры (легковесный Mozilla Minimo и не очень легковесный Firefox), Pidgin в роли Instant Messaging клиента, MPlayer с надстройкой pythm на Python и GTK+, различные мультимедиа кодеки (a52m, aalib, avcodec, faac, faad2, ffmpeg, OGG, FLAC etc), TangoGPS — GPS в связке с картами Google, в роли оконного менеджера Enlightement, разнообразные эмуляторы терминала и командные интерпретаторы (включая zsh!), кучи нативных игрушек и прочие вкусности.#
datePublished: #Mon Oct 13 15:30:51 2008#
dateUpdated: #Mon Oct 13 15:30:51 2008#
# Person begin ####################
name: #JamesBond007#
# Person end ######################
# Item end ########################
# Feed end ########################

